src/www/ui/page/UploadPermissionPage.php

```diff
@@ -75,6 +75,13 @@ protected function handle(Request $request)
     $newgroup = intval($request->get('newgroup'));
     $newperm = intval($request->get('newperm'));
     $public_perm = $request->get('public', -1);
+    $commu_status = fo_communicate_with_scheduler('status', $response_from_scheduler, $error_info);
+    if ($commu_status) {
+      $response_from_scheduler = "";
+    } else {
+      $response_from_scheduler = "Warning: Scheduler is not running!";
```

Comment:
It should be reported as error instead of warning. Warning implies if someone starts the scheduler, the changes will be applied. It is not the actual case however.